kong/plugins/hmac-auth/schema.lua

```diff
@@ -29,6 +29,7 @@ return {
               elements = { type = "string", one_of = ALGORITHMS },
               default = ALGORITHMS,
           }, },
+          { realm = { description = "When authentication fails the plugin sends `WWW-Authenticate` header with `realm` attribute value.", type = "string", required = false }, },
```

Comment:
This requires backwards-compat changes, right?

Reply:
You're right - without the backwards-compat changes the config is rejected by DP. I've created a that fixes it:
https://github.com/Kong/kong/pull/13042

@jschmid1 , @hanshuebner could you guys take a look at it?

Reply:
+ I've also added proper backwards-compat changes.